AndroidSDKCore/src/main/java/com/leanplum/internal/Log.java

```diff
@@ -63,7 +63,8 @@
      * Used only for Leanplum SDK debugging. Not visible to customers. Not sent to us when remote
      * logging is enabled.
      */
-    DEBUG
+    DEBUG,
+
```

Comment:
We don't need to commit non-functional changes. Please remove anything that isn't in scope of the feature.